docs/GOVERNANCE.md

```diff
@@ -155,6 +155,24 @@ The list can be found at [https://github.com/expressjs/discussions/blob/HEAD/doc
 
 The list can be found at [https://github.com/expressjs/discussions/blob/HEAD/docs/contributing/captains_and_committers.md#current-initiative-captains](https://github.com/expressjs/discussions/blob/HEAD/docs/contributing/captains_and_committers.md#current-initiative-captains)
 
+## Inactivity and Emeritus Policy for Any Role
+
+To support the health and continuity of the project, all individuals holding a role within the community (Triager, Committer, Project Captain, or TC member) are expected to maintain active participation.
+
+Inactivity is defined as the absence of meaningful involvement in the project—such as contributions, code reviews, triage, meeting attendance, or discussion participation—for a continuous period of 6 months.
+
+### Exceptions
+Anyone may request a temporary leave from active participation due to personal or professional reasons. In such cases, the individual should inform the relevant team or the Technical Committee (TC). During this time, the inactivity policy is paused, and the individual will not be flagged as inactive.
+
+### Inactivity Process
+- If no exception is communicated and inactivity is detected, the individual will be notified and invited to resume participation within a reasonable timeframe.
+- If participation is not resumed, the individual may be transitioned to an emeritus role that reflects their past contributions.
```

Comment:
I would much prefer that we just notify them and transition them.  Adding an additional waiting period is just another thing to track.  It's also ambiguous what a "reasonable timeframe is", and that can get muddy trying to decide that a specific time period would be.  And none of that is worth it because in nearly all cases, people who have stopped participating are not suddenly going to spring into action again just because we have notified them they have been inactive.

Reply:
I agree, It is really easy to just re-add folks.

Reply:
The new wording reflects this change.